app/lint.xml

```diff
@@ -1,4 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <lint>
     <issue id="MissingTranslation" severity="ignore" />
+    <issue id="ExtraTranslation" severity="ignore" />
```

Comment:
@mikeNG would this be fine in AOSP builds or should we just delete strings here?

Reply:
We shouldn't touch translations as that can affect weblate. After this PR is merged, weblate will delete the extra translations automatically. There shouldn't be any issue with AOSP builds